packages/slice-machine/components/Navigation/VideoItem.tsx

```diff
@@ -44,7 +48,7 @@ export const VideoItem = forwardRef<HTMLLIElement, VideoItemProps>(
       >
         <SideNavListItem ref={ref}>
           <SideNavLink
-            title="Tutorial"
+            title={isNext ? "Academy" : "Tutorial"}
```

Comment:
That's a super interesting topic because with @bapmrl we agree with you 😛

But @hypervillain, I think you told us that for this video tooltip we should not rely on a hook and that's the responsibility of Slice Machine UI. Correct?

Reply:
Yes. Of course if it gets unmanageable I would reconsider it, but I don't think Marketing links should be part of adapters, for plenty of reasons

Reply:
I can see a world where this makes some sense for marketing copy OK.

Maybe scoping it under some kind of general explicit hook/helper would at least help make it clearer and scale better.

(example hook)
```typescript
// src/hooks/useMarketingCopy.ts
import { useAdapterName } from "./useAdapterName";

// Not sure if we have one of these somewhere already
type Framework = "next" | "nuxt" | "nuxt2" | "sveltekit";

type UseMarketingCopyArgs<T> = [specificCopy: Record<Framework, T>, fallbackCopy: T];

export function useMarketingCopy<T>(...args: UseMarketingCopyArgs<T>): T {
	const [specificCopy, fallbackCopy] = args;

	const adapterName = useAdapterName();
	const framework= adapterName.split("/").pop() as string;

	return framework in specificCopy ? specificCopy[framework] : fallbackCopy;
}
```

(example usage)
```ts
// packages/slice-machine/components/Navigation/VideoItem.tsx
const copy = useMarketingCopy(
	{
		next: { title: "Academy", video: "https://example.com/next", url: "https://prismic.io/academy/prismic-and-nextjs" }
	},
	{ title: "Tutorial", video: "https://example.com/prismic", url: "https://prismic.io/docs" },
);
```

Reply:
I really like it as a middle ground. I will use your proposal 🙂

Reply:
(might need some tweaking as I just wrote that there, but cool!)

Reply:
Done 🙂
It should be easy with my proposal to drop and replace by an adapter hook one day if we feel it's too complicated to maintain in Slice Machine UI.